### PR TITLE
Refresh the Test Replay page

### PR DESCRIPTION
**Pull request by jennifer-shehane**
Addresses https://github.com/cypress-io/cypress-services/issues/9512

### CODE REVIEW

File: docs/cloud/features/test-replay.mdx [hunk 1]
```diff
@@ -12,71 +12,89 @@ sidebar_position: 40
...
-### Test Replay vs Screenshots and Videos
+## Key Benefits
 
-Before Cypress `v13`, test failures in CI have historically been captured through screenshots, videos, and stack trace outputs, but these artifacts provide limited information. Developers often need more context to understand the root cause of failures, requiring them to reproduce the issue locally, which may not always be possible due to environmental or network conditions. This knowledge gap between test failures and debugging capabilities leads to increased debugging time and potentially deploying unresolved bugs to production.
+Test failures in CI have historically been captured through screenshots, videos, and stack trace outputs,
+but these artifacts provide limited information. Developers often need more context to understand the
+root cause of failures, requiring them to reproduce the issue locally, which may not always
+be possible due to environmental or network conditions.
+This knowledge gap between test failures and debugging capabilities leads to increased debugging
+time and potentially deploying unresolved bugs to production.
```
Comment (ryanthemanuel) [inline docs/cloud/features/test-replay.mdx:29]: The transition into the starred list seems a little abrupt. Maybe an intro to the list of some kind?

```suggestion
time and potentially deploying unresolved bugs to production. Test Replay solves these problems by providing the following features.
```

or something

File: docs/cloud/features/test-replay.mdx [hunk 2]
```diff
@@ -12,71 +12,89 @@ sidebar_position: 40
...
 
-## Debugging With Test Replay
+### 3. Debug With Test Replay
 
 Both options above display the Test Replay interface, which we'll unpack from top to bottom in more detail below.
 
-### Run header
+#### Run header
 
 The left side of the header displays the Cypress Cloud test run ID number (#), with the Git branch name along with the spec, platform and browser information. Hovering on this exposes additional test meta data.
```
Comment (ryanthemanuel) [inline docs/cloud/features/test-replay.mdx:99]:
```suggestion
The left side of the header displays the Cypress Cloud test run ID number (#), with the Git branch name along with the spec, platform and browser information. Hovering on this exposes additional test metadata.
```

File: docs/cloud/features/test-replay.mdx [hunk 3]
```diff
@@ -114,6 +132,111 @@ Toggle each test attempt, play back errors at a variety of speeds or scrub throu
...
+Currently Test Replay does not support the following:
+
+- Tests recorded in Cypress App below v13.
+- WebKit browser (Safari) ([see issue](https://github.com/cypress-io/cypress/issues/28895))
+- Firefox browser ([see issue](https://github.com/cypress-io/cypress/issues/28894))
+- Video ([see issue](https://github.com/cypress-io/cypress/issues/29903))
+- Audio ([see issue](https://github.com/cypress-io/cypress/issues/29905))
+- Objects that do not have `type="image/svg+xml"` ([see issue](https://github.com/cypress-io/cypress/issues/29904))
+- Shadow DOM with `slotAssignment="manual"` ([see issue](https://github.com/cypress-io/cypress/issues/29107))
+- Shadow DOM canvas ([see issue](https://github.com/cypress-io/cypress/issues/30438))
```
Comment (ryanthemanuel) [inline docs/cloud/features/test-replay.mdx:166]:
```suggestion
- Shadow DOM canvases ([see issue](https://github.com/cypress-io/cypress/issues/30438))
```

File: docs/cloud/features/test-replay.mdx [hunk 4]
```diff
@@ -114,6 +132,111 @@ Toggle each test attempt, play back errors at a variety of speeds or scrub throu
...
+- Server sent events ([see issue](https://github.com/cypress-io/cypress/issues/29908))
+- Web sockets ([see issue](https://github.com/cypress-io/cypress/issues/29907))
+- localStorage & sessionStorage ([see issue](https://github.com/cypress-io/cypress/issues/29909))
+- Cookies ([see issue](https://github.com/cypress-io/cypress/issues/29910))
+- Console props of Cypress commands outside of `cy.request()` ([see issue](https://github.com/cypress-io/cypress/issues/29911))
```
Comment (ryanthemanuel) [inline docs/cloud/features/test-replay.mdx:171]: Do we want to call out that we do not capture network traffic of `cy.request()` either?

File: docs/cloud/features/test-replay.mdx [hunk 5]
```diff
@@ -114,6 +132,111 @@ Toggle each test attempt, play back errors at a variety of speeds or scrub throu
...
+#### Recording
+
+Capturing this additional data can use additional resources while recording, but you can
+expect an even performance exchange with Test Replay versus video recording.
+There is more activity to capture Test Replay data but without the overhead of recording,
+compressing, and storing video assets.
+
+:::info
+
+**Tip:** We recommend disabling video recording when recording with
```
Comment (AtofStryker) [inline docs/cloud/features/test-replay.mdx:184]: is it clear that video is disabled by default?

**Review by AtofStryker — APPROVED** [review]

File: docs/cloud/features/test-replay.mdx (hunk 5 above)
Comment (jennifer-shehane) [reply docs/cloud/features/test-replay.mdx:184]: There are so many customers I've seen that have it on, I do think it needs to be called out very explicitly.

File: docs/cloud/features/test-replay.mdx (hunk 4 above)
Comment (jennifer-shehane) [reply docs/cloud/features/test-replay.mdx:172]: Addressed here [`7ec343a` (#5985)](https://github.com/cypress-io/cypress-documentation/pull/5985/commits/7ec343a678e0c8096d598249fe0f73823421f37a) I created a public issue to track this also.

File: docs/cloud/features/test-replay.mdx (hunk 3 above)
Comment (jennifer-shehane) [reply docs/cloud/features/test-replay.mdx]: Updated here to be more specific on plurality and elements vs other things [`56e77c8` (#5985)](https://github.com/cypress-io/cypress-documentation/pull/5985/commits/56e77c8547c7ac7c4ac3752e6e44a73170bfc15c)

**Review by ryanthemanuel — APPROVED** [review]

**Pull request merged**